pkg/api/errors/state.go

```diff
@@ -0,0 +1,114 @@
+/*
+Copyright 2023 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package errors
+
+import (
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"google.golang.org/grpc/codes"
+
+	"github.com/dapr/kit/errors"
```

Comment:
for consistency, maybe consider making all the kit errors imports be `kitErrors`. I went with this approach in my draft PR. that way there are 2 distinct names that are also standard and easily identifiable: `kitErrors` and `apiErrors`
```suggestion
	kitErrors "github.com/dapr/kit/errors"
```